WordPress/Classes/Models/ThemeIdHelper.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+@objc (ThemeIdHelper)
```

Comment:
Because you're subclassing `NSObject`, the class will be implicitly `@objc` – so you can remove this line :)